docs/resources/securityhub_hub.md

```diff
@@ -10,7 +9,28 @@ description: |-
 
 The AWS::SecurityHub::Hub resource represents the implementation of the AWS Security Hub service in your account. One hub resource is created for each Region in which you enable Security Hub.
 
+## Example Usage
 
+### Basic Usage
+
+```terraform
+resource "awscc_securityhub_hub" "example" {
+  tags = [{

```

Comment:
`awscc_securityhub_hub` is one of the resources which has an issue on tags schema. Here it is a map of string compared to the attributes list we have on others. I would suggest removing the tags on this resource for the timebeing.

Reply:
@quixoticmonk Removed and updated the commit, thanks.

(P.S. I typically amend the commit and force-push to keep things cleaner. If you prefer additional commits in the future, please let me know.)

Reply:
Either one works as these would be squashed before merging. Appreciate doing the changes.